src/layouts/EditContactUs.jsx

```diff
@@ -0,0 +1,703 @@
+              [destination.index, 0, elem], // Splice elem into its new position
+            ],
+          },
+        });
+        const elemError = errors.locations[source.index];
+        newErrors = update(errors, {
+          locations: {
+            $splice: [
+              [source.index, 1], // Remove elem from its original position
+              [destination.index, 0, elemError], // Splice elem into its new position
+            ],
+          },
+        });
+        const elemDisplay = displaySections.locationCardsDisplay[source.index];
+        newDisplaySections = update(displaySections, {
+          locationCardsDisplay: {
+            $splice: [
+              [source.index, 1],
+              [destination.index, 0, elemDisplay],
+            ],
+          },
+        });
+        break;
+      }
+      default: {
+      }
+    }
+
+    this.setState({
+      frontMatter: newFrontMatter,
+      errors: newErrors,
+      displaySections: newDisplaySections,
+    });
+  }
+
+  onFieldChange = async (event) => { 
+    try {
+      const { state } = this;
+      const { frontMatter, footerContent } = state
+      const { errors } = state;
+      const { id, value } = event.target;
+      const idArray = id.split('-');
+      const elemType = idArray[0];
+
+      let newFrontMatter, newFooterContent, newErrors;
+      switch (elemType) {
+        case 'feedback': {
+          newFooterContent = update(footerContent, {
+            [elemType]: {$set: value},
+          });
+          break;
+        }
+        case 'header': {
+          const field = idArray[1]; 
+          
+          newFrontMatter = update(frontMatter, {
+            [field]: {$set: value},
+          });
+          break;
+        }
+        case 'contact': {
+          const contactIndex = parseInt(idArray[1], RADIX_PARSE_INT);
+          const contactType = idArray[2]; 
+          const contentIndex = parseInt(idArray[3], RADIX_PARSE_INT);
+
+          switch (contactType) {
+            case 'title':
+              newFrontMatter = update(frontMatter, {
+                contacts: {[contactIndex]: {[contactType]: {$set: value }}},
+              });
+              newErrors = update(errors, {
+                contacts: {[contactIndex]: {[contactType]: {$set: validateContact(contactType, value)}}}
+              })
+              break;
+            default: // 'phone', 'email', 'other'
+              newFrontMatter = update(frontMatter, {
+                contacts: {[contactIndex]: {content : {[contentIndex]: {[contactType]:  {$set: value } }}}},
+              });
+              newErrors = update(errors, {
+                contacts: {[contactIndex]: {content : {[contentIndex]: {[contactType]:  {$set: validateContact(contactType, value) } }}}},
+              });
+              break;
+          }
+          break;
+        }
+        case 'location': { 
+          const { locations } = state.frontMatter;
```

Comment:
Unused line of code

Reply:
this has been fixed in the latest branch of code https://github.com/isomerpages/isomercms-frontend/tree/feat/contact-us-page-2